### PR TITLE
Graphics subdialog work

### PR DESCRIPTION
**Pull request by maxwellfundi**
Added front end for the xaxis and legend.

### CODE REVIEW

File: instat/sdgPlots.Designer.vb [hunk 1]
```diff
@@ -71,9 +71,26 @@ Partial Class sdgPlots
...
+        Me.CheckBox7 = New System.Windows.Forms.CheckBox()
+        Me.TextBox2 = New System.Windows.Forms.TextBox()
+        Me.TextBox3 = New System.Windows.Forms.TextBox()
+        Me.CheckBox8 = New System.Windows.Forms.CheckBox()
+        Me.CheckBox6 = New System.Windows.Forms.CheckBox()
+        Me.grpYScale = New System.Windows.Forms.GroupBox()
+        Me.TextBox4 = New System.Windows.Forms.TextBox()
+        Me.TextBox5 = New System.Windows.Forms.TextBox()
+        Me.CheckBox9 = New System.Windows.Forms.CheckBox()
+        Me.CheckBox10 = New System.Windows.Forms.CheckBox()
```
Comment (dannyparsons) [inline instat/sdgPlots.Designer.vb]: All these controls need correct names.

**Comment by dannyparsons** [comment]
I think there should be an InitlaiseDialog sub in here like we have on other dialogs.

File: instat/sdgPlots.vb [hunk 2]
```diff
@@ -26,6 +28,9 @@ Public Class sdgPlots
         ucr2ndFactorReceiver.SetDataType("factor")
         autoTranslate(Me)
 
+        grpLabels.Visible = False
+        grpTitle.Visible = False
```
Comment (dannyparsons) [inline instat/sdgPlots.vb]: These are defaults so they don't belong in Load.
But also, the visible status should be controlled by some radio button/check box on the dialog. So you should set the radio button/check box status of what is controlling these and not the visible property directly.

**Comment by maxwellfundi** [comment]
Am closing this because i have changed the code on and submitted updated version of this

**Comment by dannyparsons** [comment]
Ok

**Pull request closed without merging**